PathPlanner/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required( VERSION 2.8 )
+
+set( CMAKE_BUILD_TYPE "Debug" )
+
+project( polysync-path-planner-algorithm-cpp )
```

Comment:
Is it common to have two projects in one CMake file? How does it determine the project name at the bottom of the file with the TARGETS variable?